### PR TITLE
Upping default dispatcher parallelism to reduce 503/504 errors

### PR DESCRIPTION
**Pull request by dasmb**

### CODE REVIEW

File: api/conf/base.conf [hunk 1]
```diff
@@ -27,6 +27,15 @@ apibuilder.sendErrorsTo="mbryzek@alum.mit.edu"
 
 git.version=0.13.0
 
+default-dispatcher {
```
Comment (gheine) [inline api/conf/base.conf]: Does this have to be in the akka actor namespace? See https://www.playframework.com/documentation/2.6.x/ThreadPools

**Comment by mbryzek** [comment]
nice!  @dasmb want to see gregor's comment? and ping if you need me to merge (can't recall if you have merge privilege here)

File: api/conf/base.conf (hunk 1 above)
Comment (dasmb) [reply api/conf/base.conf]: Yes absolutely

**Comment by dasmb** [comment]
I have merge privilege!

**Pull request merged**